lib/fastlane/actions/crashlytics.rb

```diff
@@ -43,7 +43,7 @@ def self.run(params)
 
       def self.assert_params_given!(params)
         return unless params.empty?
-        raise 'You have to pass Crashlytics parameters to the Crashlytics action, take a look at https://github.com/KrauseFx/fastlane#crashlytics'.red
+        fail 'You have to pass Crashlytics parameters to the Crashlytics action, take a look at https://github.com/KrauseFx/fastlane#crashlytics'.red
```

Comment:
Line is too long. [149/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.